### PR TITLE
remove spurious print statements

### PR DESCRIPTION
**Pull request by mdehoon**
This PR removes some spurious print statements, and cleans up some doctests.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Bio/ExPASy/cellosaurus.py [hunk 1]
```diff
@@ -18,20 +18,23 @@
...
-    ...        if 'Homo sapiens' in record['OX'][0]:
-    ...            print(record['ID'])
+This example downloads the Cellosaurus database and parses it:
+
+    >>> from urllib.request import urlopen
+    >>> from io import TextIOWrapper
+    >>> from Bio.ExPASy import cellosaurus
+    >>> url = "ftp://ftp.expasy.org/databases/cellosaurus/cellosaurus.txt"
+    >>> bytestream = urlopen(url)
+    >>> textstream = TextIOWrapper(bytestream, "UTF-8")
```
Comment (peterjc) [inline Bio/ExPASy/cellosaurus.py:28]: Ugly, but I can't immediately think of anything shorter.

Worth a tiny bit of explanation in the comment above?

File: Bio/SwissProt/KeyWList.py [hunk 2]
```diff
@@ -85,7 +84,7 @@ def parse(handle):
             elif key in ("DE", "SY", "GO", "HI", "WW"):
                 record[key].append(value)
             else:
-                print(f"Ignoring: {line.strip()}")
+                raise ValueError(f"Cannot parse line '{line.strip()}'")
```
[the remark this answers is not in the document]
Comment (mdehoon) [reply Bio/SwissProt/KeyWList.py:87]: There is only a single `keywlist.txt` file provided by uniprot. This warning didn't happen in the past and it doesn't happen with the latest version of `keywlist.txt`.

File: Bio/ExPASy/cellosaurus.py (hunk 1 above)
Comment (mdehoon) [reply Bio/ExPASy/cellosaurus.py:32]: I added a comment.

**Review by peterjc — APPROVED** [review]

**Comment by mdehoon** [comment]
Thanks!

**Comment by peterjc** [comment]
I logged #4199 for the GitHub Actions Docs failure which started last week.

**Pull request merged**